product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -6,15 +6,86 @@ The database parameters listed on the DB Configuration tab are also referred to
+- For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+
+-  For more information on the types of values parameters accept, see [Parameter Names and Values](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). The types apply to both PostgreSQL and EDB Postgres Advanced Server parameters.
+
 
 !!!note
 Not all database configuration parameters are supported by BigAnimal. Some parameters, such as `wal_level` and `restore_command`, are reserved for EDB to provide the managed database features of BigAnimal. 
 !!!
 
+## Using formulas for parameter values
+
+In addition to entering specific values for parameters, for some parameters you can specify formulas to automatically calculate a value. You can use formulas for parameters of type integer and real using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`. Use `?` amd `:` in ternary formulas such as the [shared buffer example](#example). Use `( )` to specify [order of operations](order-of-operations), if needed. GUCs used in formulas must also be of type integer or real. All arithmetic is done on 64-bit floating point values rounded to an integer result.  
+
+BigAnimal has what we refer to as psuedo GUCs to facilitate creating equations. These read-only GUCs are:
+- `cpu_cores` &#8212 number of cores provided by the VM running the cluster nodes
+- `disk_iops` &#8212 number of IOPS allocated to the disks backing the primary data volume
+- `disk_size` &#8212 amount of memory backing the primary data volume, in KB
+- `ram` &#8212 amount of memory provided by the VM running the cluster nodes, in KB
+- `replica_count` &#8212 total number of nodes in the cluster
+
+Formulas can calculate a value relative to the specifications you selected for the cluster or relative to other non-pseudo GUCs. If you resize your cluster, BigAnimal recalculates formulas using the new values for the pseudo GUCs. 
+
+### Order of operations
+`*` and `/` are at the same operator precedence. `+` and `-` are at the same operator precedence. `*` and `/` are evaluated before `+` and `-`. Use parentheses to specify additional order of operations. 
+
+### Units
+GUCs can either be a bare number, such as 8, or have a unit, such as GB. Units are categorized into classes such as size or duration.
+
+You can't use GUCs with different classes of units in formulas. For example, you can't use a formula that tries to multiply two seconds with three kilobytes (2s \* 3KB), but you can use a formula that multiplies two sizes even if they have different scales (2KB \* 8GB). 
+
```

Comment:
Let's get rid of those backslashes.

Reply:
They don't show up in the output and if I don't escape the asterisks it causes italics. 

![image](https://user-images.githubusercontent.com/83650384/176928994-41acbbb5-895b-4171-944b-c387a9a60249.png)